conftest.py

```diff
@@ -215,31 +220,29 @@ def can_connect_to_mongo():
 
 
 @pytest.fixture(scope='function', params=_all_databases)
-def db_type(request):
+def db_type(request, db_name):
 
     db_list = request.config.option.test_databases
     if request.param not in db_list and 'all' not in db_list:
         pytest.skip("Skipping {} DB, set --test-all-databases=True".format(request.param))
 
     # If testing mongo, make sure we can connect, otherwise skip.
-    if request.param == 'mongo' and not can_connect_to_mongo():
+    if request.param == 'mongo' and not can_connect_to_mongo(db_name):
         pytest.skip("Can't connect to {} DB, skipping".format(request.param))
-    PanDB.permanently_erase_database(
-        request.param, 'panoptes_testing', really='Yes', dangerous='Totally')
+    PanDB.permanently_erase_database(request.param, db_name, really='Yes', dangerous='Totally')
     return request.param
 
 
 @pytest.fixture(scope='function')
-def db(db_type):
+def db(db_type, db_name):
     return PanDB(
-        db_type=db_type, db_name='panoptes_testing', logger=get_root_logger(), connect=True)
+        db_type=db_type, db_name=db_name, logger=get_root_logger(), connect=True)
 
 
 @pytest.fixture(scope='function')
-def memory_db():
-    PanDB.permanently_erase_database(
-        'memory', 'panoptes_testing', really='Yes', dangerous='Totally')
-    return PanDB(db_type='memory', db_name='panoptes_testing')
+def memory_db(db_name):
```

Comment:
Why does it make sense to add db_name as a positional param to this fixture?

Reply:
You mean for the `memory_db` in particular? I suppose it is not necessary as it will never persist beyond the tests but it was mostly just for the consistency of having all the db types having the same name that is easily changed in one place. Can remove from here.